playbooks/roles/topbeat/tasks/main.yml

```diff
@@ -0,0 +1,34 @@
+---
+- name: Download topbeat 1.0.0
```

Comment:
This shouldn't be hard-coded.

Tangentially, I don't think we even need this level of specificity. I
know this pattern is cargo-culted throughout this repository [1]:
- Download {{package}} {{version}}
- Install {{package}}
- Enable {{package}} at boot
- etc...

and while we do want to convey these ideas, it's often an unnecessary
customization that leads to maintenance issues.

What usually happens (and maybe happened here?), is that someone
copy/pastes an older task list, then replaces `s/old/new/`. But humans
being humans, we often overlook/mistake some so you end up seeing things
mistakenly hard-coded.

Or maybe the role gets updated in the future and while updating all lines, you miss a few.

Is this the biggest problem we face? Of course not.
But, at least in this case, it's a completely superfluous complication.
The point of the "name" field is to provide context while the steps are
executed; "what's it doing now?".
But as you've noticed, ansible already tells us the role to which the
task belongs:
```
[role: topbeat] "Download thing"
# or some variation thereof
```

So in most cases, we're only adding more work for ourselves [2], while
risking confusion [3], for no added benefit:

```
[role: topbeat] "Download topbeat package"
```

is no more explanatory than

```
[role: topbeat] "Download package"
```

in this context; it's obvious what we're doing. Natually, added
specificity does help at times, like:

```
[role: topbeat] "Download python dependency"
# instead of
[role: topbeat] "Download package"
```

So you don't _have_ to change the wording everywhere per above, but at the very least, we shouldn't hard-code versions :)

[1] https://en.wikipedia.org/wiki/Cargo_cult_programming
[2] Whether you search/replace, edit the copy/paste, or type it all out manually
[3] It's one thing to mistakenly say "restart filebeat" inside the topbeat role; we can probably intuit the intent. But if it says "restart logstash" or "restart supervisor" in the topbeat role, it's possible that you actually meant that, instead of being a glaring typo.